src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingService.kt

```diff
@@ -117,4 +135,15 @@ class VurderingService(private val behandlingService: BehandlingService,
         return OppdaterVilkår.erAlleVilkårsvurderingerOppfylt(lagredeVilkårsvurderinger)
     }
 
+    fun utledBarnId(barnId: UUID?,
+                    alleBarnPåForrigeBehandling: List<BehandlingBarn>,
+                    alleBarnPåGjeldendeBehandling: List<BehandlingBarn>): UUID? =
+            barnId?.let { barnIdPåForrigeVurdering ->
+                val barnFraTidligereVurdering = alleBarnPåForrigeBehandling.firstOrNull { it.id == barnIdPåForrigeVurdering }
+                val barnForGjeldendeVurdering = alleBarnPåGjeldendeBehandling.firstOrNull {
+                    (barnFraTidligereVurdering?.personIdent != null && it.personIdent == barnFraTidligereVurdering.personIdent) ||
+                    (barnFraTidligereVurdering?.søknadBarnId != null && it.søknadBarnId == barnFraTidligereVurdering.søknadBarnId)
+                }
+                barnForGjeldendeVurdering?.id
+            }
```

Comment:
Blir dette riktig, for alle tilfeller?

Reply:
Hva skjer hvis et barn bytte personIdent?
Hva skjer dersom et barn ikke har personIdent og IKKE er koblet til en søknad?